aiormq/base.py

```diff
@@ -137,10 +137,8 @@ async def close(
         if self.is_closed:
             return None
 
-        await asyncio.wait_for(
-            self.loop.create_task(self.__closer(exc)),
-            timeout=timeout,
-        )
+        countdown = Countdown(timeout)
```

Comment:
Is `Countdown` just a sugar for `asyncio.wait_for` with support of async context management?

Reply:
nope, it's single timeout for multiple actions

Reply:
I mean you can create some inner function with all these actions and await it with timeout using `asyncio.wait_for`.